client/components/Draft.jsx

```diff
@@ -63,7 +63,7 @@ export default function Draft(props) {
       .catch(() => {
         setLoading(false);
       });
-  }, []);
+  });
```

Comment:
put this back in, we don't want to trigger the effect on every rerender
to fix the lint error you were getting previously, add `props` and `refsEditor` to the array, so that the second argument is `[props, refsEditor]`

Reply:
oh oops idk why i deleted it lol